src/frontend/src/service_config/grpc.rs

```diff
@@ -26,6 +27,8 @@ pub struct GrpcOptions {
     pub max_recv_message_size: ReadableSize,
     // Max gRPC sending(encoding) message size
     pub max_send_message_size: ReadableSize,
+    #[serde(default = "Default::default")]
```

Comment:
Can you add the configuration in `config/frontend.example.toml` and `standalone.example.toml` and execute `make config-docs`?